src/components/menus/content-contextual-menu.js

```diff
@@ -586,6 +586,7 @@ const ContentContextualMenu = (props) => {
                 icon: <FileCopyIcon fontSize="small" />,
             });
         }
+
```

Comment:
useless line break which this component modified by this commit but it's not.

Reply:
That's the most important part of this PR in fact, because in this code block, that was the only place without a line break :)
Other line breaks like this missing one are on lines 518, 527, 538, 548, 565, 576, 600 and 612.